.storybook/themes/news.scss

```diff
@@ -1788,6 +1788,46 @@
 				quad-chain-child-item-empty: (
 					display: contents,
 				),
+				results-list: (
+					components: (
+						attribution: (
+							margin-top: var(--global-spacing-4),
+						),
+						media-item: (
+							float: right,
+							max-width: 100px,
+						),
+						heading: (
+							font-size: var(--heading-level-5-font-size),
+							font-weight: var(--heading-level-5-font-weight),
+							line-height: var(--heading-level-5-line-height),
+							margin-bottom: var(--global-spacing-4),
+						),
+						separator: (
+							padding: 0 var(--global-spacing-2),
+						),
+					),
+				),
+				results-list-container: (
+					padding-top: var(--global-spacing-4),
+				),
+				results-list-container-first: (
+					padding-top: 0,
+				),
+				results-list-show-image: (
+					components: (
+						heading: (
+							width: 68%,
+						),
+					),
+				),
+				results-list-see-more: (
+					components: (
+						button: (
+							margin-top: var(--global-spacing-4),
+						),
+					),
+				),
```

Comment:
This is not exposed by your SCSS.  It's also not needed with the change I will recommend